packages/common/src/ui/components/popover/BasePopover/BasePopover.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { FC } from 'react';
+import Fade from '@mui/material/Fade';
+import Paper, { PaperProps } from '@mui/material/Paper';
+import Popper, { PopperProps } from '@mui/material/Popper';
+
+export interface BasePopoverProps extends Omit<PopperProps, 'open'> {
+  // I like having is as a prefix for boolean props
+  isOpen: boolean;
+  anchorEl: PopperProps['anchorEl'];
+  paperProps?: PaperProps;
+}
+
+export const BasePopover: FC<BasePopoverProps> = ({
+  children,
+  isOpen,
+  anchorEl,
+  paperProps,
+  ...popperProps
+}) => {
+  return (
+    <Popper
+      {...popperProps}
+      open={isOpen}
+      anchorEl={anchorEl}
+      transition
+      placement="bottom-start"
+    >
+      {({ TransitionProps }) => (
+        <Fade {...TransitionProps} timeout={350}>
+          <Paper
+            {...paperProps}
+            sx={{
+              width: 240,
```

Comment:
I like having the dimensions as props, since they are forced here: currently to override you have to set the `sx` and it isn't clear that this is required. having explicit dimension props would help, I think

Reply:
nice! 
7ecb0b7